gempython/tools/amc_user_functions_xhal.py

```diff
@@ -0,0 +1,343 @@
+import os, signal, sys
+sys.path.append('${GEM_PYTHON_PATH}')
```

Comment:
this is not compatible

Reply:
though i see it currently exists in other places and just needs to be removed...

Reply:
Okay, so I understand that all instances of:

```
sys.path.append('${GEM_PYTHON_PATH}')
```

Should be removed.  Correct?

Reply:
roger, but i suppose this will render this current code inoperable, as per `rw_reg` business...